lecture_02/assignment_01/Etienne_Pavoncello/assignment_01.py

```diff
@@ -0,0 +1,32 @@
+from compas.artists import Artist
+from compas.datastructures import Mesh
+from compas.geometry import Box
+from compas.geometry import Frame
+from compas.geometry import Plane
+from compas.geometry import Projection
+from compas.geometry import Point
+from compas.geometry import Vector
+
+point = Point(15, 15, 15)
+xaxis = Vector(1, 0, 0)
+yaxis = Vector(0, 1, 10)
+frame = Frame(point,xaxis,yaxis)
```

Comment:
Code style: add whitespace after commas:
```suggestion
frame = Frame(point, xaxis, yaxis)
```